src/Nethermind/Nethermind.Baseline/BaselineModule.cs

```diff
@@ -96,7 +96,7 @@ public async Task<ResultWrapper<Keccak>> baseline_insertCommit(Address address,
             tx.GasLimit = 1000000;
             tx.GasPrice = 20.GWei();
 
-            Keccak txHash = await _txSender.SendTransaction(tx, TxHandlingOptions);
+            (Keccak txHash, AddTxResult? addTxResult) = await _txSender.SendTransaction(tx, TxHandlingOptions);
```

Comment:
if you are not using the value use either:

(Keccak txHash, _) = await _txSender.SendTransaction(tx, TxHandlingOptions);

Or

Keccak txHash, = (await _txSender.SendTransaction(tx, TxHandlingOptions)).Hash;